nvflare/fuel/hci/client/config.py

```diff
@@ -44,8 +44,10 @@ def __init__(self, workspace: Workspace):
             sys.path.append(custom_dir)
 
         admin_config_file_path = workspace.get_admin_startup_file_path()
+        config_files = [admin_config_file_path]
         resources_file_path = workspace.get_resources_file_path()
-        config_files = [admin_config_file_path, resources_file_path]
+        if resources_file_path:
+            config_files.append(resources_file_path)
```

Comment:
Make it optional looks good to me.

@nvkevlu and @IsaacYangSLA do we have place (AdminAPI or Provision) that required this resource file?

Reply:
Not required by AdminAPI side at least, so looks good to me

Reply:
Provision does not generate resource files for admin startup kits.  That is, admin startup kit's local folder is empty.  However, provision still created that empty folder because I remember admin failed to start if it couldn't find that folder.

Reply:
This PR will make that resources folder as an optional.